src/dmd/lexer.d

```diff
@@ -3389,6 +3228,121 @@ private struct TimeStampInfo
     }
 }
 
+private enum LS = 0x2028;       // UTF line separator
```

Comment:
Redundant private.

Reply:
I know. But I wanted to minimize diffs when doing massive code motion.